configure.py

```diff
@@ -2253,7 +2253,7 @@ def make_bin_override():
 gyp_args += ['-Dpython=' + python]
 
 if options.use_ninja:
-  gyp_args += ['-f', 'ninja-' + flavor]
+  gyp_args += ['-f', 'ninja-' + flavor, '-G', 'config=' + config['BUILDTYPE']]
```

Comment:
Is it only applying to ninja but not other toolchains?

Reply:
Yes, both `make` and msvc generators only generate one build type at a time.